src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs

```diff
@@ -20,14 +22,26 @@ public void Configure(EndpointConfiguration endpointConfiguration)
         // https://docs.particular.net/transports/sqs/configuration-options
         var transport = endpointConfiguration.UseTransport<SqsTransport>();
 
+        NServiceBus.Logging.LogManager.GetLogger(nameof(AmazonSQSProfile)).Info(RegionEndpoint.GetBySystemName(cs["Region"].ToString()).DisplayName);
+
+        var credentials = new EnvironmentVariablesAWSCredentials();
+        var region = RegionEndpoint.GetBySystemName(cs["Region"].ToString());
```

Comment:
Why not first extract the region by name, then log the display name? So essentially refactor it to following:

```
var region = RegionEndpoint.GetBySystemName(cs["Region"]);
LogManager.GetLogger(nameof(AmazonSQSProfile)).Info(region.DisplayName);
// rest
```